doc/api.md

```diff
@@ -64,14 +82,14 @@ for f in files:
 %sql sqlite:///db_one.db
 ```
 
-## List connections
-
-Connect to another database to demonstrate `--list`'s usage:
+Assign an alias to the connection (**added 0.5.2**):
```

Comment:
Why not -n /--name ?

Reply:
we already have a `-n`/`--no-index` option

Reply:
Sad, ok makes sense.